google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -89,6 +90,13 @@ public String getAppProfileId() {
     return getTypedStubSettings().mutateRowSettings();
   }
 
+  /** Returns the object with the settings used for BulkMutations. */
+  public BatchingCallSettings<RowMutation, Void> bulkMutationsSettings() {
+    // NOTE: the user facing BigtableDataClient only exposes this api as newBulkMutations,
+    // so the outer name should match that. While the stub sticks to rpc names.
```

Comment:
s/should match that. While the stub sticks to/should match that, while the stub should stick to/ (and same down below)

Reply:
done